hubot-slack/Dockerfile

```diff
@@ -8,6 +8,8 @@ USER hubot
 
 WORKDIR /home/hubot
 
+RUN npm install underscore
```

Comment:
Maybe this can be installed in one step with the rest of the packages (see line 4)?